From 8c2d31f5a4a4e1e39c8ad2e4e997ebcf0b32c1c6 Mon Sep 17 00:00:00 2001
From: Swaroop Balan <swaroop.balan@intel.com>
Date: Thu, 23 Aug 2018 20:38:22 +0530
Subject: [PATCH] [CELADON] Change api level to 25

Compare to OMR1 gordon-peak, it should be 26.
Due to meet a build error from making the sepolicy.

Change the first api level to 25 at first.

Tracked-On: None
Signed-off-by: Swaroop Balan <swaroop.balan@intel.com>
---
 cel_apl/cel_apl.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/cel_apl/cel_apl.mk b/cel_apl/cel_apl.mk
index 5cafdaf..b2dc22e 100644
--- a/cel_apl/cel_apl.mk
+++ b/cel_apl/cel_apl.mk
@@ -9,3 +9,5 @@ PRODUCT_BRAND := cel_apl
 PRODUCT_DEVICE := cel_apl
 PRODUCT_MODEL := Generic cel_apl
 PRODUCT_MANUFACTURER := Intel
+
+PRODUCT_SHIPPING_API_LEVEL := 25
-- 
1.9.1

